api/envoy/config/core/v3/protocol.proto

```diff
@@ -633,6 +633,11 @@ message Http2ProtocolOptions {
   // If unset, HTTP/2 codec is selected based on envoy.reloadable_features.http2_use_oghttp2.
   google.protobuf.BoolValue use_oghttp2_codec = 16
       [(xds.annotations.v3.field_status).work_in_progress = true];
+
+  // [#not-implemented-hide:] Hiding until Envoy has full metadata support.
```

Comment:
? Didn't you just implement it? Can you also add a release note?

Reply:
The `allow_metadata` option, which is required for this to work is similarly hidden. It seemed reasonable to me that this should be hidden if other HTTP/2 metadata options are hidden as well. What do you think?